pymc3/distributions/continuous.py

```diff
@@ -146,8 +146,8 @@ def random(self, point=None, size=None, repeat=None):
     def logp(self, value):
         lower = self.lower
         upper = self.upper
-        return bound(-tt.log(upper - lower),
-                     value >= lower, value <= upper)
+        return bound_elemwise(-tt.log(upper - lower),
```

Comment:
shouldn't this require `ones_like` also, if we are going that route to deal with the broadcasting issue?